### PR TITLE
Story

### PR DESCRIPTION
**Pull request by bentleybiscuit**
Hi Charlie - please code review!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!-- DOCTYPE html -->
```
Comment (charliecog) [inline index.html]: this shouldnt be a comment

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+		<link rel="stylesheet" type="text/css" href="normalize.css">
+		<link rel="stylesheet" type="text/css" href="styles.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		
+		<title>Charlotte Nightingale portfolio</title>
+	</head>
+	
+	<body>
+		<nav>
+			<div>
```
Comment (charliecog) [inline index.html]: remove extra div

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+			</div>			
+		</nav>
+		<header>
+			<section class="container">
+				<h1>Charlotte Nightingale</h1>
+				<h2>full stack web developer</h2>
+			</section>
+		</header>
+		
+			<div class="portfolio" id="portfolio">
```
Comment (charliecog) [inline index.html]: indentation

File: index.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+	
+			
+			</div>
+			
+			<section class="container">
+				<h1 id="aboutMe">About me:</h1>
+				<p>This is where I'm going to tell you all about myself and the skills that I am developing on the Full Stack Track at Mayden Academy.  There's the minor detail of the online test, and then I'll be a certified scrum master, which I'll tell you about too.  I'm based in Bath and am looking for my first role as a junior developer: I'm available from the middle of December 2019.</p>
+			</section>
+			
+			<footer>
```
Comment (charliecog) [inline index.html]: need a closing footer tag too

File: navbarBuild.html [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<!-- DOCTYPE html -->
```
Comment (charliecog) [inline navbarBuild.html:1]: remove file if not using

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+	width: 33%;
+	float: left;
+	background-repeat: no-repeat;
+	background-size: auto;
+	background-position: center;
+}
+
+
+.portfolio #projectOne {
+	background-image: url("https://cdn.pixabay.com/photo/2019/08/29/19/48/cobweb-4439844__340.jpg");
```
Comment (charliecog) [inline styles.css]: save images locally so you dont have to rely on that website being live for ever

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+
+
+.container h1 {
+	color: #019dcf;
+	font-size: 20px;
+	font-weight: 400;
+	letter-spacing: .05em;
+	margin: 30px;
+	
+
```
Comment (charliecog) [inline styles.css]: remove extra whitespace

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+	
+}
+
+footer p {
+	color: #019dcf;
+	font-size: 20px;
+	font-weight: 400;
+	margin: 30px;
+	text-align: center;
+	letter-spacing: 0.05em;
```
Comment (charliecog) [inline styles.css]: stick with rem if you used them elsewhere

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+	font-weight: 400;
+	letter-spacing: .05em;
+	margin: 30px;
+	
+	
+}	
+
+.container p {
+	color: #019dcf;
+	font-size: 20px;
```
Comment (charliecog) [inline styles.css:90]: use rems if you used them elsewhere

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+	font-weight: 400;
+
+}
+
+
+@media screen and (max-width: 1220px){
+	nav {
+			font-size: .95rem;
```
Comment (charliecog) [inline styles.css]: indentation

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+		font-size: .95rem;
+	}
+	
+	.portfolio .project {
+		width: 100%;
+		font-size: .95rem;
+
+	}
+
+
```
Comment (charliecog) [inline styles.css]: whitespace

File: styles.css [hunk 12]
```diff
@@ -0,0 +1,172 @@
...
+	}
+}
+	
+	
+
+@media screen and (min-width: 500px){
+	.portfolio .project {
+		height: 400px;
+	}
+
```
Comment (charliecog) [inline styles.css]: whitespace

File: styles.css [hunk 13]
```diff
@@ -0,0 +1,172 @@
...
+}
+
```
Comment (charliecog) [inline styles.css]: remove whitespace line

File: styles.css [hunk 14]
```diff
@@ -0,0 +1,172 @@
...
+body {
+	font-family: arial, sans-serif;
+	
+}
+
+nav {
+	background-color: #019dcf;
+	color: #ffffff;
```
Comment (charliecog) [inline styles.css:16]: not doing anything

File: styles.css [hunk 15]
```diff
@@ -0,0 +1,172 @@
...
+	font-size: 20px;
+	font-weight: 400;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 16]
```diff
@@ -0,0 +1,172 @@
...
+	letter-spacing: .05em;
+	padding: 40px 60px;
+}
+
+nav ul {
+	margin: 0;
+	width: 100%;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 17]
```diff
@@ -0,0 +1,172 @@
...
+nav li:hover {
+	color: black;
+}	
+
+li a {
+	color: #ffffff;
+}
+
+header {
+	background-color: #ffffff;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 18]
```diff
@@ -0,0 +1,172 @@
...
+	color: #019dcf;
+	font-size: 40px;
+	font-weight: 400;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 19]
```diff
@@ -0,0 +1,172 @@
...
+	letter-spacing: .05em;
+	padding: 10px 20px;
+	text-align: center;
```
Comment (charliecog) [inline styles.css:44]: not doing anything

File: styles.css [hunk 20]
```diff
@@ -0,0 +1,172 @@
...
+.portfolio {
+	margin: 20px 20px;
+	overflow: auto;
+}
+
+.portfolio .project {
+	background-color: #019dcf;
+	color: #ffffff;
+	font-size: 20px;
+	font-weight: 400;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 21]
```diff
@@ -0,0 +1,172 @@
...
+	letter-spacing: .05em;
+	padding: 40px 60px;
+	width: 33%;
+	float: left;
+	background-repeat: no-repeat;
+	background-size: auto;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 22]
```diff
@@ -0,0 +1,172 @@
...
+	font-weight: 400;
+	letter-spacing: .05em;
+	margin: 30px;
+	
+}
+
+footer p {
+	color: #019dcf;
+	font-size: 20px;
+	font-weight: 400;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 23]
```diff
@@ -0,0 +1,172 @@
...
+	letter-spacing: .05em;
+	margin: 30px;
+	
+	
+}	
+
+.container p {
+	color: #019dcf;
+	font-size: 20px;
+	font-weight: 400;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 24]
```diff
@@ -0,0 +1,172 @@
...
+	margin: 30px;
+	text-align: center;
+	letter-spacing: 0.05em;
+}
+
+footer p .email {
+	color: #019dcf;
+	font-size: 20px;
```
Comment (charliecog) [inline styles.css:105]: not doing anything

File: styles.css [hunk 25]
```diff
@@ -0,0 +1,172 @@
...
+	font-weight: 400;
```
Comment (charliecog) [inline styles.css]: not doing anything

**Review by charliecog — COMMENTED** [review]
general comments:

- margin: 0 on the body will get rid of white gutter around the edge

Looks good, really clean build - almost there!

File: navbarBuild.html (hunk 5 above)
Comment (bentleybiscuit) [reply navbarBuild.html:1]: which file please?

**Pull request merged**